### ISSUE TITLE
Selected and not selected.

### ISSUE THREAD

**Issue by zirouan** [igalata/Bubble-Picker#48]
Hello.

Is it possible to tell background of the circle and color of the text for selecting and not selected?

Ex:

I want all the balls to be white and the text color red.
When selecting, I want the selected ball to be in red and the text in white.

Is that possible?

Thank you.

**Comment by rubinbasha** [comment]
Would be nice to have some more options in the library, options like: 
- setBubbleSize to be dynamic depending on the text that is in the bubble
- setTextColorSelected
- setTextColorUnselected
- setBackgroundDrawableSelected
- setBackgroundDrawableUnselected
- setBubbleExpandFactorOnSelect (where expand factor 0f should also be accepted)

**Comment by lazy-coder-10** [comment]
Yes, you can change text color and item color in Item.kt file. Let me know if you need an example here is the sample which I have customized on this library.
![bubble picker](https://user-images.githubusercontent.com/13958364/110744527-d4e1bd80-825f-11eb-9168-73365b56bd5a.jpeg)

**Comment by deshmukhdhiraj** [comment]
@lazy-coder-10 can you please provide sample code for the same

**Comment by vtproduction** [comment]
> Yes, you can change text color and item color in Item.kt file. Let me know if you need an example here is the sample which I have customized on this library. ![bubble picker](https://user-images.githubusercontent.com/13958364/110744527-d4e1bd80-825f-11eb-9168-73365b56bd5a.jpeg)

@lazy-coder-10 could you provide some sample code for changing the text color, please!

**Comment by lazy-coder-10** [comment]
you can make the following changes  on current libs 

`if(withImage){
            drawText(canvas, pickerItem.selectedTextColor ?: Color.BLACK)
        }else{
            drawText(canvas, pickerItem.textColor ?: Color.WHITE)
        }
`
